<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage</class>
 <widget class="QMainWindow" name="Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>University Facilities Management System (UFMS)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="oframe4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="copyright">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>570</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Â© SLAUATRON 2019.</string>
    </property>
   </widget>
   <widget class="Line" name="oframe2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="oframe1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="oframe3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="currentuser_lbl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Session:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameuser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_newbook">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>MAKE A NEW BOOKING</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewbook">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>MANAGE BOOKING(S)</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>761</width>
       <height>271</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>561</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_return">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>440</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RETURN 
SELECTION</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>781</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_returnall">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>500</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>RETURN ALL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>380</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Double-Click the box on the left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RETURN</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>781</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BOOKING(S)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>400</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Selection: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>400</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameadmin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_viewava">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>VIEW AVAILABILITY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewbooked">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>VIEW ALL BOOKED 
VENUE/FACILITIES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_updatestatus">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>UPDATE STATUS OF
 VENUE/FACILITIES</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameupdatestatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="tablestatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>761</width>
       <height>271</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QListView" name="listViewstatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>371</y>
       <width>561</width>
       <height>181</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BOOKING(S)</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelselectedstatus">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>400</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>380</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Double-Click the box on the left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_update">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>460</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>APPROVE 
REQUEST</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>400</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Selection: </string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>781</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RETURN</string>
     </property>
    </widget>
   </widget>
   <zorder>oframe4</zorder>
   <zorder>copyright</zorder>
   <zorder>oframe2</zorder>
   <zorder>oframe1</zorder>
   <zorder>oframe3</zorder>
   <zorder>currentuser_lbl</zorder>
   <zorder>userLabel</zorder>
   <zorder>btn_logout</zorder>
   <zorder>frame1</zorder>
   <zorder>frameadmin</zorder>
   <zorder>frameuser</zorder>
   <zorder>frameupdatestatus</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
